src/sentry/api/endpoints/organization_integrations.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
+
+from rest_framework import serializers
+from rest_framework.response import Response
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.provider import ProviderSerializer
+from sentry.exceptions import PluginError
+from sentry.plugins import bindings
+
+
+class IntegrationSerializer(serializers.Serializer):
+    providerId = serializers.CharField(max_length=64, required=True)
+    defaultAuthId = serializers.CharField(max_length=64, required=False)
+    # TODO(jess): actually make this work with gh integrations
+    integrationId = serializers.CharField(max_length=64, required=False)
+
+    def validate(self, attrs):
+        if not (attrs.get('defaultAuthId') or attrs.get('integrationId')):
+            raise serializers.ValidationError(
+                'You must either provide a defaultAuthId or an integrationId'
+            )
+        return attrs
+
+
+class OrganizationIntegrationsEndpoint(OrganizationEndpoint):
```

Comment:
could we put a feature detect test in the post/get endpoints just so we dont have users trying to mess with things before this is in a better state